### ISSUE TITLE
aes(stroke) -> css stroke-width

### ISSUE THREAD

**Issue by tdhock** [tdhock/animint#127]
This is a new aesthetic in ggplot2 v2.0 that we can use to render the css stroke-width property

```
geom_point() gains a stroke aesthetic which controls
 the border width of shapes 21-25 (#1133, @SeySayux).
 size and stroke are additive so a point with size = 5 and 
stroke = 5 will have a diameter of 10mm. (#1142)
```

https://github.com/hadley/ggplot2/blob/master/NEWS.md#ggplot2-200

**Comment by tdhock** [comment]
@faizan-khan-iit I don't think this is a super important issue, but it should be an easy fix, now that you got animint working with the new ggplot2 >= 2.0.

**Comment by faizan-khan-iit** [comment]
Closed by #157
